test/wizard.jl

```diff
@@ -414,6 +414,18 @@ end
     end
 end
 
+@testset "Wizard - state serialization" begin
+    for state_generator in (Wizard.WizardState, step2_state, step3_state, step7_state)
```

Comment:
Unrelated to this PR, but I wonder if we can rename `Wizard.WizardState` to `Wizard.State`

Reply:
I think it might be confusing to see `State` objects floating around within the `Wizard` module.  More likely, we can just export `WizardState` from `Wizard`, and then `using BinaryBuilder.Wizard` in this test suite.